tests/unit/test_matrix.py

```diff
@@ -242,3 +243,16 @@ def it_knows_how_to_assemble_its_blocks(self, request):
             [bs._base_values, bs._subtotal_columns],
             [bs._subtotal_rows, bs._intersections],
         ]
+
+
+class Describe_SumSubtotals(object):
+    @pytest.mark.parametrize(("prop_name",), (("foo",), ("bar",)))
+    def it_returns_correct_base_values(self, _cube_result_matrix_prop_, prop_name):
+        ss = _SumSubtotals(_cube_result_matrix_prop_, prop_name)
+        assert ss._base_values == getattr(_cube_result_matrix_prop_, prop_name)
+
+    # fixture components ---------------------------------------------
+
+    @pytest.fixture
+    def _cube_result_matrix_prop_(self, request):
+        return property_mock(request, Assembler, "_cube_result_matrix")
```

Comment:
and this will need to become an instance mock. There is no `._cube_result_matrix` property on `_SumSubtotals` I don't think. It's an instance variable isn't it?